### ISSUE TITLE
ARM applications force-close

### ISSUE THREAD

**Issue by Black-Seraph** [Black-Seraph/android_patches_gpdwin#3]
If you find yourself in the situation that your ARM-only application force-closes on you, you can try enabling the v8_y version of houdini by executing the enable_nativebridge command in a su-elevated Terminal while being connected to WiFi, and then restarting your affected application.

This won't always solve the issue, but does so in most lightweight ARM-based applications.

Due to the fact that we have to rely on Intel-provided blobs to improve our ARM application compatibility this issue is marked as wontfix, as there is literally nothing we can do to improve compatibility further other than wait on Intel to release new versions of the houdini libraries.
